src/licensedcode/data/rules/nysl-0.9982.RULE

```diff
@@ -0,0 +1 @@
+http://www.kmonos.net/nysl/index.ja.html
```

Comment:
Why do we have a URL here ?

Reply:
IIRC, we explicitly have the text in the `.RULE` file. You can have a look at the other files for reference. :)

Reply:
i looked at the other `.RULE` files they all had the URL of the license homepage.. :)

Reply:
Can you point me to a few of them ?

Reply:
`bigdigits_1.RULE`, `biopython.RULE`, `attribution.RULE`, `artistic-2.0_2.RULE`. Some of them also contain the license text, but most of them contains the URL.

Reply:
Thanks @techytushar  👍

Reply:
:+1: